includes/class-wcs-payment-tokens.php

```diff
@@ -47,6 +47,27 @@ public static function update_subscription_token( $subscription, $new_token, $ol
 		$subscription->update_meta_data( $token_meta_key, $new_token->get_token() );
 		$subscription->save();
 
+		$available_gateways        = WC()->payment_gateways->get_available_payment_gateways();
+		$new_token_payment_gateway = $new_token->get_gateway_id();
+
+		// Add the new token to the subscription.
+		if ( isset( $available_gateways[ $new_token_payment_gateway ] ) ) {
+			$gateway_instance = $available_gateways[ $new_token_payment_gateway ];
+			$gateway_instance->add_token_to_order( $subscription, $new_token );
```

Comment:
The `$gateway->add_token_to_order()` is only found on the WCPay gateway (not on Stripe or Square for example), so I don't think we can use this here inside WC Subscriptions Core.